devicetypes/smartthings/zigbee-plugin-motion-sensor.src/zigbee-plugin-motion-sensor.groovy

```diff
@@ -23,7 +23,7 @@ metadata {
         capability "Health Check"
         capability "Sensor"
        
-        fingerprint profileId: "0104", deviceId: "0107", inClusters: "0000, 0003, 0004, 0406", outClusters: "0006, 0019", model: "E280-KR0A0Z0-HA", deviceJoinName: "Smart Occupancy Sensor (AC Type)"
+        fingerprint profileId: "0104", deviceId: "0107", inClusters: "0000, 0003, 0004, 0406", outClusters: "0006, 0019", model: "E280-KR0A0Z0-HA", deviceJoinName: "Motion Sensor" //Smart Occupancy Sensor (AC Type)
```

Comment:
I believe this is an `eZEX` brand device. The `messages.properties` file will need to be updated